### ISSUE TITLE
Models' Logits return, 

### ISSUE THREAD

**Issue by ajam74001** [takuseno/d3rlpy#278]
Hi @takuseno, I was wondering how to return the logits (estimated Q values for each action) of CQL model (or any other Q based learners).

Cheers, 
Many thanks for your help in advance,

**Comment by takuseno** [comment]
@ajam74001 Thanks for the issue. Unfortunately, there is no function by default. But, you can do it by accessing to internal objects:
```
# just example
# dqn needs to be built before this.
dqn = d3rlpy.algos.DQN()
dqn.impl.q_function(torch.rand((1, 10)))
```

**Comment by ajam74001** [comment]
Great! Many thanks for the response, dear @takuseno.
